hpy/universal/src/hpymodule.c

```diff
@@ -12,6 +12,10 @@
 #include "handles.h"
 #include "common/version.h"
 
+#ifdef PYPY_VERSION
```

Comment:
I'm wondering what we should do about other cases -- e.g. this should likely also error on GraalPython, but probably not error on Pyston.

Reply:
I fear the only option is to add it case-by-case.
Note that for PyPy it's just a sanity check: if you compile it it works, but it's clearly not what you want.